Algorithm-Everything-Be-True.md

```diff
@@ -24,7 +37,7 @@ You could use loops or callbacks functions, there are multiple ways to solve thi
 
 **Solutions ahead!**
 
-## Code Solution using for-in loop & hasOwnProperty:
+## :beginner: Beginner : Code Solution using for-in loop & hasOwnProperty:
```

Comment:
~~## :beginner: Beginner : Code Solution using for-in loop & hasOwnProperty:~~ ## :beginner: Beginner Code Solution (using for-in loop & hasOwnProperty):